include/CL/sycl/buffer/detail/accessor.hpp

```diff
@@ -429,10 +429,10 @@ class accessor : public detail::debug<accessor<T,
 
     /* Create the OpenCL buffer and copy in data from the host if in
        read mode */
-    cl_buf = { task->get_queue()->get_boost_compute().get_context(),
-               get_size(),
-               flags,
-               is_read_access() ? array.data() : 0 };
+    cl_buf = boost::compute::buffer{ task->get_queue()->get_boost_compute().get_context(),
```

Comment:
the secret goal of triSYCL is to push the limit of post-modern C++ too... :-)
I guess the correct way for flaky compiler is to use
```
    cl_buf.emplace_assign(task->get_queue()->get_boost_compute().get_context(),
                          get_size(),
                          flags,
                          is_read_access() ? array.data() : 0);
```